modules/mssql/mssql.go

```diff
@@ -0,0 +1,98 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const defaultImage = "mcr.microsoft.com/mssql/server:2022-latest"
+
+const (
+	defaultPort     = "1433/tcp"
```

Comment:
Port 1433 is the typical port for mssql - [source](https://learn.microsoft.com/en-us/sql/sql-server/install/configure-the-windows-firewall-to-allow-sql-server-access?view=sql-server-ver16)